### PR TITLE
add volume size parameters

### PR DESCRIPTION
**Pull request by richm**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -219,6 +219,18 @@ of literal identifiers to use:
 
     $ oct configure aws-defaults master_subnet_ids subnet-XXXXXXXX,subnet-XXXXXXXX,subnet-XXXXXXXX
 
+The root volume size to use for a master (default: 35) can be configured with:
```
Comment (stevekuznetsov) [inline README.rst]: `s/size/size in GB/`

File: README.rst [hunk 2]
```diff
@@ -219,6 +219,18 @@ of literal identifiers to use:
...
+
+.. code-block:: shell
+
+    $ oct configure aws-defaults master_root_volume_size NUMBER_OF_GB
+
+The container volume size to use for a master (default: 35) can be configured with:
```
Comment (stevekuznetsov) [inline README.rst]: `s/size/size in GB/`

**Comment by richm** [comment]
You are correct - it works for the root volume but not for the container volume - with settings of 64 for both:
```
$ df -h 
Filesystem                                   Size  Used Avail Use% Mounted on
/dev/xvda2                                    64G   16G   49G  24% /
devtmpfs                                     7.8G     0  7.8G   0% /dev
tmpfs                                        7.8G     0  7.8G   0% /dev/shm
tmpfs                                        7.8G  3.6M  7.8G   1% /run
tmpfs                                        7.8G     0  7.8G   0% /sys/fs/cgroup
/dev/mapper/docker-openshift--xfs--vol--dir   11G   33M   11G   1% /mnt/openshift-xfs-vol-dir
```
So - should I back out that change? I'm mostly concerned with the root volume, but since there was a setting for the `volume_size` for `/dev/sdb` I went ahead and added it.

**Comment by stevekuznetsov** [comment]
You could either back out the configuration option for the other device or edit [this `if` statement](https://github.com/openshift/origin-ci-tool/blob/master/oct/cli/provision/remote/all_in_one.py#L215) to be triggered also if the storage is non-default... but I am not certain that the playbook will correctly behave to expand the existing VG/LV. So maybe configuring only the root device is the better choice here.

**Comment by richm** [comment]
Can this be merged?  Anything else required?  I've been using this for a couple of weeks now and it works well.

**Comment by stevekuznetsov** [comment]
Sorry, didn't see that you had updated this. LGTM, thanks for the update @richm :)

**Pull request merged**